frontend/src/scenes/ingestion/v2/panels/PanelComponents.tsx

```diff
@@ -54,3 +58,47 @@ export function PanelHeader(): JSX.Element | null {
         </div>
     )
 }
+
+export function DemoProjectButton({ text, subtext }: { text: string; subtext?: string }): JSX.Element {
+    const { next } = useActions(ingestionLogicV2)
+    const { createTeam } = useActions(teamLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { updateCurrentTeam } = useActions(userLogic)
+    const { reportIngestionTryWithDemoDataClicked, reportProjectCreationSubmitted } = useActions(eventUsageLogic)
+
+    const demoTeamName: string = 'Demo'
```

Comment:
If there already is the "Demo" label/snack, then it seems quite redundant to name the project "Demo" too. Why not use the "Hedgebox" name?

BTW since this is an app-level constant, it should probably be at the top level of the file. Conventionally would be called `DEMO_TEAM_NAME`.